src/builtin-addons/core/template-definition-provider.ts

```diff
@@ -192,10 +192,9 @@ export default class TemplateDefinitionProvider {
       paths = [...getPathsForComponentScripts(root, maybeComponentName), ...getPathsForComponentTemplates(root, maybeComponentName)].filter(fs.existsSync);
     }
 
-    if (!paths.length) {
-      paths = mAddonPathsForComponentTemplates(root, maybeComponentName);
```

Comment:
looks like it should be before addon name check

Reply:
see https://github.com/lifeart/ember-language-server/pull/214#discussion_r606098365,
we could adjust `if` here,
basically, without batman syntax is should work as is, with batman, we could remove `if` here and just add results from it's call